### PR TITLE
Fixes eemepmodelrunner to look for volcanoes and npps at the same time

### PR DESCRIPTION
**Pull request by dulte**

### CODE REVIEW

File: utils/SnapPy/eemepModelRunner [hunk 1]
```diff
@@ -134,6 +138,7 @@ class EemepModelRunner():
                 print(ifile, sys.stderr)
             return
         if (len(volcanoes) == 0):
+            print("Could not find any volcanoes or npps")
```
Comment (magnusuMET) [inline utils/SnapPy/eemepModelRunner:141]: Remove print statement

File: utils/SnapPy/eemepModelRunner [hunk 2]
```diff
@@ -146,11 +151,15 @@ class EemepModelRunner():
             except:
                 fh. write("  started by: unknown\n")
 
+
+        xml_filename = Path(infile).name
+
         try:
             self._log(dirpath, 'Starting run for: {}'.format(infile))
-            mr = ModelRunner(dirpath, hpc, npp)
+            run_npp = True if xml_filename == self.npp_xml_filename else False
```
Comment (magnusuMET) [inline utils/SnapPy/eemepModelRunner:159]: `run_npp = xml_filename == self.npp_xml_filename`

File: utils/SnapPy/eemepModelRunner [hunk 3]
```diff
@@ -175,15 +184,15 @@ if __name__ == '__main__':
     parser.add_argument("--dir", help="top-level dir to search for volcano.xml/npp.xml files", required=True)
     parser.add_argument("--dir2", help="second top-level dir, main-dir if --dir is not writable")
     parser.add_argument("--hpc", help="HPC-machine to run job on", required=True)
-    parser.add_argument("--npp", action="store_true", help="simulates a nuclear accident instead of volcano")
+    # parser.add_argument("--npp", action="store_true", help="simulates a nuclear accident instead of volcano")
```
Comment (magnusuMET) [inline utils/SnapPy/eemepModelRunner:187]: Remove

File: utils/SnapPy/Snappy/EEMEP/resources/job_script_ppi_centos7_direct_npp.job [hunk 4]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 ## {runtag} {rundir} {year}-{month}-{day}T{hour} {runhour}
 
-#$ -N eemep_nuc
+#$ -N eemep
```
Comment (magnusuMET) [inline utils/SnapPy/Snappy/EEMEP/resources/job_script_ppi_centos7_direct_npp.job:4]: Is this name used by the script to locate the right job?

File: utils/SnapPy/eemepModelRunner (hunk 2 above)
Comment (heikoklein) [reply utils/SnapPy/eemepModelRunner:159]: Please use brackets here, to make operator precedence clearer.

File: utils/SnapPy/Snappy/EEMEP/resources/job_script_ppi_centos7_direct_npp.job (hunk 4 above)
Comment (dulte) [reply utils/SnapPy/Snappy/EEMEP/resources/job_script_ppi_centos7_direct_npp.job:4]: I think(!) it the name given to the eemep run on PPI

Comment (heikoklein) [reply utils/SnapPy/Snappy/EEMEP/resources/job_script_ppi_centos7_direct_npp.job:4]: This name is only used by humans during `squeue`. Both eemep and eemep_nuc are fine with me.

**Pull request merged**